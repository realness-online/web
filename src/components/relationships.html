<template>
  <section id="relationships">
    <nav v-for="profile in following" itemprop="following" >
      <profile-as-uid></profile-as-uid>
    </nav>
    <nav v-for="profile in blocked" itemprop="blocked">
      <profile-as-uid></profile-as-uid>
    </nav>
  </section>
</template>
