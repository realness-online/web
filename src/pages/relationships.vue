<template>
  <section id="relationships">
    <header>
      <h1>Relationships</h1>
    </header>
  </section>
</template>
<style lang='stylus'>
  @require '../style/variables'
  section#relationships
    padding: 0 base-line
</style>
