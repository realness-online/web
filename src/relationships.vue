<template lang="html">
  <section id="relationships">
    <header>
      <h1>Relationships</h1>
    </header>
  </section>
</template>

<style lang="stylus">
</style>

<script>
  export default {
    data () {
      return {
      }
    }
  }
</script>
