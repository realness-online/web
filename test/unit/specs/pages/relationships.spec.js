import Vue from 'vue'
import { shallow } from 'vue-test-utils'
import relationships from '@/pages/relationships'

describe('pages/relationships.vue', () => {
  it('render relationship information', () => {
    let wrapper = shallow(relationships)
    expect(wrapper.element).toMatchSnapshot()
  })
})
