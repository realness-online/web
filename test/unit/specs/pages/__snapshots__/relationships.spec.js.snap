// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/pages/relationships.vue render relationship information 1`] = `
<section
  id="relationships"
>
  <header>
    <h1>
      Relationships
    </h1>
  </header>
</section>
`;
